### PR TITLE
FEPLT-2144: Add bottomStyle prop to NavigationBar

### PR DESCRIPTION
**Pull request by DSil**
This component now accepts a `bottomStyle` prop that accepts either `"shadow"` or `"border"`, to determine the bottom style of the component.

`"shadow"` is default value, keeping the visual aspect of the component unchanged.

Visual tests for the component were also added, since they were missing

### CODE REVIEW

File: packages/orbit-components/src/NavigationBar/README.md
Comment (mvidalgarcia) [inline packages/orbit-components/src/NavigationBar/README.md:1]: Totally out of scope, but we have a broken image here.

image

Perhaps something for @sarkaaa to take a look?

File: packages/orbit-components/src/NavigationBar/index.tsx [hunk 1]
```diff
@@ -63,10 +64,12 @@ const NavigationBar = ({
       data-test={dataTest}
       id={id}
       className={cx(
-        "bg-white-normal shadow-fixed p-300 z-navigation-bar fixed inset-x-0 top-0 box-border flex w-full translate-x-0 items-center",
+        "bg-white-normal p-300 z-navigation-bar fixed inset-x-0 top-0 box-border flex w-full translate-x-0 items-center",
         "duration-normal transform-gpu transition-transform ease-in-out",
         "tb:h-1600 h-[52px]", // As defined on the const above
-        shown ? "translate-y-0" : "tb:-translate-y-1600 translate-y-[-52px]", // As defined on the const above
+        shown ? "translate-y-0" : "tb:-translate-y-1600 -translate-y-xxxl", // As defined on the const above
```
Comment (mvidalgarcia) [inline packages/orbit-components/src/NavigationBar/index.tsx:70]: Out of the scope, but what const?

File: packages/orbit-components/src/NavigationBar/NavigationBar.ct.tsx-snapshots/visual-NavigationBar-NavigationBar-default-1-Medium-Mobile-linux.png
Comment (mvidalgarcia) [inline packages/orbit-components/src/NavigationBar/NavigationBar.ct.tsx-snapshots/visual-NavigationBar-NavigationBar-default-1-Medium-Mobile-linux.png:1]: lol the linux emojis are 🙈

**Review by mvidalgarcia — APPROVED** [review]
Minor comments, all of them non-blocking or out of the scope, so approving.

File: packages/orbit-components/src/NavigationBar/index.tsx [hunk 2]
```diff
@@ -63,10 +64,12 @@ const NavigationBar = ({
       data-test={dataTest}
       id={id}
       className={cx(
-        "bg-white-normal shadow-fixed p-300 z-navigation-bar fixed inset-x-0 top-0 box-border flex w-full translate-x-0 items-center",
+        "bg-white-normal p-300 z-navigation-bar fixed inset-x-0 top-0 box-border flex w-full translate-x-0 items-center",
         "duration-normal transform-gpu transition-transform ease-in-out",
         "tb:h-1600 h-[52px]", // As defined on the const above
```
[the remark this answers is not in the document]
Comment (DSil) [reply packages/orbit-components/src/NavigationBar/index.tsx:69]: Yes and no. "-xxxl" is deprecated and "form-box-large" is a component specific token, so we should not use it here. I don't know if we can tell eslint to not suggest some classes 😕

File: packages/orbit-components/src/NavigationBar/index.tsx (hunk 1 above)
Comment (DSil) [reply packages/orbit-components/src/NavigationBar/index.tsx:70]: Actually, I don't know why this change happened 🤔 As is said above, `xxxl` is deprecated. But I don't recall making this change 🤯

File: packages/orbit-components/src/NavigationBar/README.md
Comment (mvidalgarcia) [reply packages/orbit-components/src/NavigationBar/README.md:1]: I DMed Sarka, this should be treated separately, resolving.

**Pull request merged**